django/contrib/admin/static/admin/js/admin/RelatedObjectLookups.js

```diff
@@ -157,4 +157,25 @@
     window.showAddAnotherPopup = showRelatedObjectPopup;
     window.dismissAddAnotherPopup = dismissAddRelatedObjectPopup;
 
-})();
+    $(document).ready(function() {
```

Comment:
Unless I'm reading it incorrectly, we're inside `(function($)` not `$(function($)` so I think it's needed.